pkg/supportbundle/staticspecs/vendorspec.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: troubleshoot.sh/v1beta2
+kind: SupportBundle
+metadata:
+  name: cluster-specific-supportbundle
```

Comment:
Rename to `vendor-supportbundle`

Reply:
Thank you @cbodonnell, good catch! Changed